src/View/Home/country.html.twig

```diff
@@ -0,0 +1,28 @@
+{% block contryDay %}
+
+<div class="container bg-light text-dark" >
+    <div  class="card-header" align="center"><h2>COUNTRY DAY</h2></div>
+    <div class="row ">
+        <div class="col-lg-6">
+            <figure class="figure">
+{% for one in country if(one.id)==1 %}
```

Comment:
"one" ce n'est pas un bon nom, si tu suis ma recommandation plus haut, tu devrais ici avoir
`{% for countryDay in countryDays %}`
de plus, ici, tu récupères "en dur" le coutry day avec l'id 1, ce qui n'est pas une bonne chose. Ton code ne devrais jamais contenir un id en dur car si celui ci change, ton code ne fonctionnera plus. 
Par ailleurs, je ne comprends pas trop ce que tu veux faire en réalité, c'est récupérer le dernier country day ajouté ? dans ce cas, il faut que tu reprennes ta requete SQL, faire un ORDER BY date et un LIMIT 1, et retourne un fetch() plutot qu'un fetchAll()...